### PR TITLE
chore(ux): improve help messages and some logs

### PR DESCRIPTION
**Pull request by ralgozino**
- Improve help messages adding missing subcommands in the main summary and details where missing.
- Fix some outdate information in help messages.
- Improve the redaction of some of the help messages.
- Improve some log messages that were giving information that the user does not care about.

### CODE REVIEW

File: cmd/get.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 func NewGetCmd() *cobra.Command {
 	getCmd := &cobra.Command{
 		Use:   "get",
-		Short: "Get a resource (e.g. kubeconfig) from a cluster",
+		Short: "Get the kubeconfig, available upgrade paths for a cluster",
```
Comment (speziato) [inline cmd/get.go:16]:
```suggestion
		Short: "Get the kubeconfig or available upgrade paths for a cluster",
```

File: cmd/root.go [hunk 2]
```diff
@@ -190,22 +190,22 @@ func NewRootCmd() *RootCommand {
...
+		"Path where to create the data directory (.furyctl). Default is the user's home.",
 	)
 
 	rootCmd.PersistentFlags().StringVarP(
 		&rootCmd.config.Log,
 		"log",
 		"l",
 		"",
-		"Path to the log file or set to 'stdout' to log to standard output (default: ~/.furyctl/furyctl.log)",
+		"Path to the log file or set to 'stdout' to log to standard output. Default is '<outdir>/.furyctl/furyctl<timestamp>-<random number>.log'",
```
Comment (speziato) [inline cmd/root.go:201]:
```suggestion
		"Path to the log file or set to 'stdout' to log to standard output. Default is '<outdir>/.furyctl/furyctl.<timestamp>-<random number>.log'",
```

**Review by speziato — CHANGES_REQUESTED** [review]
Minor suggestions

File: cmd/root.go (hunk 2 above)
Comment (ralgozino) [reply cmd/root.go:201]: nice catch!

**Review by speziato — APPROVED** [review]
LGTM!

**Pull request merged**